lib/1.4/dml-builtins.dml

```diff
@@ -2157,7 +2157,8 @@ template bank is (object, shown_desc) {
             (start, end) = this._intersect(
                 offset, size, hits[r].offset, hits[r]._size(), true);
 
-            local uint64 r_value = 0;
+            local uint64 r_value = (r_start == 0 && r_end == hits[r]._size())
+                ? 0 : hits[r].get();
```

Comment:
We sure we want `.get()` here rather than `.get_val()`? Are we ok with the semantics that an inquiry set may incur an inquiry get -- that that's what people would expect?
I agree with the choice though. If we pick `.get_val()` we risk people overriding `get()` with their own super special semantics, and then that _silently_ won't get reflected in a partial inquiry set, leading to confusion.